ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -219,7 +222,6 @@ objects:
           --bgp-routing-mode=regional
         gcloud compute networks subnets create "${INSTANCE_PREFIX}" \
           --network "${INSTANCE_PREFIX}" \
-          --region "${GOOGLE_COMPUTE_REGION}" \
```

Comment:
so this is not needed if we specify the `compute/region` in the configuration above?

Reply:
yes, correct.